### PR TITLE
add task list,check your answers page related scripts

### PR DESCRIPTION
**Pull request by lahiruwijesingheOC**
Please review

### CODE REVIEW

File: src/test/scala/uk/gov/hmrc/test/ui/pages/CheckYourAnswersPage.scala [hunk 1]
```diff
@@ -15,5 +15,24 @@
...
+    onPage(CHECK_YOUR_ANSWERS_PAGE_TITLE)
+    isHeader(CHECK_YOUR_ANSWERS_PAGE_HEADER)
+  }
 
-object CheckYourAnswersPage extends BasePage {}
+  def verifyCheckYourAnswersPageAndContinue() = {
+    onCheckYourAnswersPage()
+    require(DataCollectorMap.checkAnswersGS == returnCheckYourAnswersPageInformation(), "Data not matching")
+    clickContinueButton()
+    //DataCollectorMap.checkAnswersGS.clear()
```
Comment (dacohm) [inline src/test/scala/uk/gov/hmrc/test/ui/pages/CheckYourAnswersPage.scala:31]: Delete this comment.

File: src/test/scala/uk/gov/hmrc/test/ui/specs/UserJourneyTests.scala [hunk 2]
```diff
@@ -465,6 +563,9 @@ class UserJourneyTests extends BaseSpec {
       When("I select any radio button and click continue")
       PercentageCausedChangeInChargPage.selectNewChargeRadioButtonAndContinue()
 
+      When("I click sign out from the page")
+      PercentageCausedChangeInChargPage.signOutPage()
+
       /*Then("I Should see the lta-protection-or-enhancements page")
       LtaProtectionOrEnhancementsPage.onLtaProtectionOrEnhancementsPage()
```
Comment (dacohm) [inline src/test/scala/uk/gov/hmrc/test/ui/specs/UserJourneyTests.scala:570]: Delete this comment.

Comment (lahiruwijesingheOC) [reply src/test/scala/uk/gov/hmrc/test/ui/specs/UserJourneyTests.scala]: Done

File: src/test/scala/uk/gov/hmrc/test/ui/pages/CheckYourAnswersPage.scala (hunk 1 above)
Comment (lahiruwijesingheOC) [reply src/test/scala/uk/gov/hmrc/test/ui/pages/CheckYourAnswersPage.scala]: Done

**Review by dacohm — APPROVED** [review]

**Pull request merged**